client/lib/app/editor/bpmn-editor.js

```diff
@@ -403,7 +403,11 @@ BpmnEditor.prototype.createModeler = function($el, $propertiesEl) {
   };
 
   var pluginModules = this.plugins.get('bpmn.modeler.additionalModules');
-
+  var moddleModules = this.plugins.get('bpmn.modeler.additionalModdles');
```

Comment:
What exactly does this line do? I.e. what is the result returned by `this.plugins.get('bpmn.modeler.additionalModdles');`?

Could you provide a test case that verifies your functionality works as expected?

Reply:
The result returned by that line should be an array of objects each containing one single moddle as expected from moddleExtensions property in BpmnJS, something like this:

```
[
{camunda: camundaModdlePackage},
{otherNamespace: otherRequiredJsonDescriptor}
]
```
and the for loop
```
for (const key of moddleModules) {
    Object.assign(moddles, key);
  }
```
is used to create a single object with every moddle registered.

I'll try to provide a test case.

Reply:
Thanks for the clarification. Please change carry out the following changes:

* Rename extension point to `bpmn.modeler.moddleExtensions`
* Instead of for of loop, please assign all extensions like this:

    ```
    var moddleExtensions = Object.assign({ camunda: ... }, ...pluginModdleExtensions);
    ```

Reply:
@nikku I've applied the changes.

Reply:
Thanks. 

> I'll try to provide a test case.

Do you still plan to follow up on the test case?

Reply:
If you need it, I can work on an example project based on [Panel property example](https://github.com/bpmn-io/bpmn-js-examples/tree/master/properties-panel-extension). With it I'll make a small test case for the functionality.

Reply:
Hello @nikku , as I said yesterday I completed now a small plugin example based on Panel property example with a small mocha test case; You can find it [here](https://github.com/sharedchains/panel-properties-extended-example). If it's not the test case you expected, please let me know and I'll try to provide something else.
Thank you!